bigquery/deploy/deploy-indexer.sh

```diff
@@ -18,15 +18,21 @@ fi
 dataset=$1
 project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
 
+# Need to get cluster name by sorting the list of clusters, and choosing to
+# use the one with the greatest timestamp (most recent)
+cluster_line=$(gcloud container clusters list | grep elasticsearch-cluster- | sort -rn -k1 | head -n1)
```

Comment:
opt if you do `gcloud help container clusters list`, you'll see there's a `sort-by` option you can use

Reply:
I would've liked to do it that way, but I couldn't configure the sort such that 'elasticsearch-cluster' comes after 'elasticsearch-cluster-{TIMESTAMP}' . We can change this to use `--sort-by` once we no longer have any clusters named 'elasticsearch-cluster`.

Reply:
Oh that makes sense, can you add a comment then?

Reply:
You merged, n/m about the comment, no big deal :)